### PR TITLE
remove V1 case block mock

### PR DESCRIPTION
**Pull request by snopoke**
Nowhere was intentionally using V1 case blocks so I decided to remove them.

@millerdev 

cc @czue @dannyroberts 

See https://github.com/dimagi/commcare-hq/pull/8687 re test failures

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 10 errors:

* corehq/apps/hqcase/tests/test_case_sharing.py, line 66 - E501 line too long (123 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 26 - E501 line too long (207 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 48 - E501 line too long (207 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 119 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 120 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 121 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 122 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 126 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 129 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/xform.py, line 24 - F401 'legacy_soft_assert' imported but unused

**Comment by millerdev** [comment]
Reviewed, but not sure I have enough context to merge, and not sure what your intent is with the failing tests.

**Comment by snopoke** [comment]
@millerdev I want @czue to take a look a the test failures (see https://github.com/dimagi/commcare-hq/pull/8687)

**Comment by dimagimon** [comment]
There are 10 errors:

* corehq/apps/hqcase/tests/test_case_sharing.py, line 66 - E501 line too long (123 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 26 - E501 line too long (207 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 48 - E501 line too long (207 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 119 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 120 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 121 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 122 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 126 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 129 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/xform.py, line 24 - F401 'legacy_soft_assert' imported but unused

**Comment by dimagimon** [comment]
There are 10 errors:

* corehq/apps/hqcase/tests/test_case_sharing.py, line 66 - E501 line too long (123 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 26 - E501 line too long (207 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 48 - E501 line too long (207 > 115 characters)
* corehq/ex-submodules/casexml/apps/case/mock.py, line 119 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 120 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 121 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 122 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 126 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/mock.py, line 129 - E241 multiple spaces after ':'
* corehq/ex-submodules/casexml/apps/case/xform.py, line 24 - F401 'legacy_soft_assert' imported but unused

File: corehq/ex-submodules/casexml/apps/case/xform.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
     ReconciliationError,
 )
 from django.conf import settings
-from couchforms.util import is_deprecation
+from couchforms.util import is_deprecation, legacy_soft_assert
```
Comment (czue) [inline corehq/ex-submodules/casexml/apps/case/xform.py]: should this be removed?

**Comment by czue** [comment]
nice to see this!

**Comment by proteusvacuum** [comment]
:tada:

**Comment by dannyroberts** [comment]
:goberserk:

File: corehq/ex-submodules/casexml/apps/case/xform.py (hunk 1 above)
Comment (snopoke) [inline corehq/ex-submodules/casexml/apps/case/xform.py]: https://github.com/dimagi/commcare-hq/pull/8716

**Comment by benrudolph** [comment]
big :moneybag:

**Pull request merged**